CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - --user and --mailbox flags have been removed from CLI examples for details and restore commands (they were already not supported, this only updates the docs).
 - Improve restore time on large restores by optimizing how items are loaded from the remote repository.
 - Remove exchange item filtering based on m365 item ID via the CLI.
+- OneDrive backups no longer include a user's non-default drives.
```

Comment:
This was never the case to begin with actually

Reply:
Kind of a regression, really.  It wasn't the case to begin with.  Then for sharepoint compliance we put in a generic drive enumerator that, we assumed, would only pick up the "one" oneDrive, opening us up to the issue.  So it's been this way for some time now, even in versions before we officially released sharepoint.